projects/ngrx.io/content/guide/migration/v16.md

```diff
@@ -21,6 +21,169 @@ Version 16 has the minimum version requirements:
 
 ## Breaking changes
 
-### TBD
 
+## Breaking changes
```

Comment:
The `Breaking changes` title is added twice

Reply:
Woops, seems like there are some issues here 😅
Thanks for the review, I'll get these tomorrow

Reply:
It should be better now